src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -2341,19 +2370,20 @@ private class ProductChangeDTO
 
         private class OrderDTO
         {
-            [FromRoute]
+            [FromRoute(Name = "id")]
```

Comment:
Please undo changes to existing models / tests.

Reply:
ok.